client/dashboard/profile-wizard/steps/theme/index.js

```diff
@@ -89,6 +90,11 @@ class Theme extends Component {
 		this.setState( { demo: theme } );
 	}
 
+	skipStep() {
+		recordEvent( 'storeprofiler_store_theme_skip_step' );
```

Comment:
Might be worthwhile to track the active theme when skipped so we can see if there's a pattern associated with skipped themes.